pkg/operator/staticpod/controller/guard/guard_controller.go

```diff
@@ -107,16 +108,30 @@ func nodeConditionFinder(status *corev1.NodeStatus, condType corev1.NodeConditio
 	return nil
 }
 
+func isNodeSchedulable(node *corev1.Node) bool {
```

Comment:
I hope this is the only and the best way to validate a node is schedulable :)

Reply:
I can not guarantee this is the best way. `oc adm cordon NODE` taints a node with `NoSchedule:node.kubernetes.io/unschedulable` taint so checking for the taint is a minimal condition. `isNodeSchedulable` name might not be the best name in this context. What about renaming the function to `nodeHasUnschedulableTaint`?

Reply:
+1 for renaming `nodeHasUnschedulableTaint`